tests/panoramic/cli/mapper/test_model.py

```diff
@@ -1,11 +1,11 @@
 import pytest
 
-from panoramic.cli.mapper import (
+from panoramic.cli.model.client import Model, ModelField
+from panoramic.cli.model_mapper import (
     map_columns_to_model,
     map_model_from_local,
     map_model_from_remote,
 )
-from panoramic.cli.model.client import Model, ModelField
 from panoramic.cli.pano_model import PanoModel, PanoModelField
```

Comment:
This file should be in `tests/panoramic/cli/test_model_mapper.py`

Reply:
Forgot to move it thanks.

Reply:
dd3be040aef4b3a787ce1945889d19de6edf7fe3